Script/item.dat

```diff
@@ -1104,8 +1104,8 @@ potion /* materialcontainer-> */
   FormModifier = 25;
   NameSingular = "bottle";
   MainMaterialConfig == GLASS;
-  SecondaryMaterialConfig = { 11, WATER, HEALING_LIQUID, OMMEL_URINE, POISON_LIQUID, VALDEMAR, ANTIDOTE_LIQUID, VODKA, TROLL_BLOOD, OMMEL_SWEAT, OMMEL_TEARS, SULPHURIC_ACID; }
-  MaterialConfigChances = { 11, 75, 50, 5, 50, 1, 10, 5, 25, 5, 5, 25; }
+  SecondaryMaterialConfig = { 13, WATER, HEALING_LIQUID, OMMEL_URINE, POISON_LIQUID, VALDEMAR, ANTIDOTE_LIQUID, VODKA, TROLL_BLOOD, OMMEL_SWEAT, OMMEL_TEARS, SULPHURIC_ACID, MUSTARD_GAS_LIQUID, LIQUID_DARKNESS; }
```

Comment:
Ommel liquid spawn rate is decreased by `x/(x+256)`, where x is the total weight of the new liquids. Not too big a difference with `x` reduced from 20 to 7. If even that is too much, the 7 points could be taken from other less useful liquids such as water, poison, vodka, or acid. Alternatively, it could offset the increased science talk opportunities.